src/main/java/benben/Storage.java

```diff
@@ -0,0 +1,119 @@
+package benben;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * The type Storage that stores, reads and rite the local file
+ */
+public class Storage {
+    private final String filePath;
+
+    private File file;
+
+    /**
+     * Instantiates a new Storage.
+     *
+     * @param filePath the file path of the local file
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Writes the data from the task list to the local file
+     *
+     * @param tasks the task list
+     */
+    public void write(TaskList tasks) {
+        try {
+            FileWriter fw = new FileWriter(this.filePath);
+            String content = "";
+            for (int i = 0; i < tasks.size(); i++) {
+                Task t = tasks.get(i);
+                content = content + t.getLog();
+            }
+            fw.write(content);
+            fw.close();
+        } catch (IOException e) {
+            throw new BenBenException("Failed to write to file!" + e.getMessage());
+        }
+    }
+
+    /**
+     * Load array list of tasks from the local file
+     *
+     * @return the array list of the tasks
+     * @throws BenBenException the ben ben exception
+     */
+    public ArrayList<Task> load() throws BenBenException {
```

Comment:
I am not sure about the length of this method. Perhaps you can shorten this method?